README.md

```diff
@@ -5,17 +5,28 @@ Blaze4D is a Fabric mod that changes Minecraft's rendering engine to use the Vul
 Early Development and is **NOT** intended for use by the faint-hearted. Support for Blaze4D can be found in the #support
 Discord channel.
 
-Note: Sodium, Iris or Canvas probably don't work yet, so don't bother trying. We're trying to add support for them ourselves
+## Things that don't work
+- Sodium, Iris or Canvas probably don't work yet, so don't bother trying. We're trying to add support for them ourselves
+- Fancy graphics
```

Comment:
Fabulous is the only one which doesnt work.

Reply:
Fixed